<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gérer les représentations</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Ajout d'une représentation</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Choisir le nom du spectacle, la date (précise à l'heure) et le taux de promotion de la représentation à ajouter.</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="comboBox_ajout_fct_comp_7">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateTimeEdit" name="dateTimeEdit">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="minimumSize">
             <size>
              <width>125</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>125</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Ajouter</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Suppression d'une représentation</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <italic>false</italic>
           </font>
          </property>
          <property name="text">
           <string>Choisir le nom du spectacle et la date de la représentation à supprimer.</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QComboBox" name="comboBox_suppr_fct_comp_7">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_suppr_fct_comp_7_2">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="minimumSize">
             <size>
              <width>125</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>125</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Supprimer</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="table_fct_comp_7">
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>nomSpec</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>dateRep</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>promoRep</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>nbPlacesDispoRep</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_fct_comp_7">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>comboBox_suppr_fct_comp_7</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>Dialog</receiver>
   <slot>refreshRepr_suppr_List()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>ajouter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>663</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>718</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>suppr()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>571</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>672</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>refreshRepr_suppr_List()</slot>
  <slot>ajouter()</slot>
  <slot>suppr()</slot>
 </slots>
</ui>
